moveit_kinematics/kdl_kinematics_plugin/CMakeLists.txt

```diff
@@ -1,14 +1,30 @@
 set(MOVEIT_LIB_NAME moveit_kdl_kinematics_plugin)
 
-add_library(${MOVEIT_LIB_NAME}
+add_library(${MOVEIT_LIB_NAME} SHARED
   src/kdl_kinematics_plugin.cpp
   src/chainiksolver_vel_mimic_svd.cpp)
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 
-target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES})
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+  rclcpp
+  random_numbers
+  pluginlib
+  moveit_core
+  orocos_kdl
+  kdl_parser
+  EIGEN3
+)
 
-install(TARGETS ${MOVEIT_LIB_NAME} 
-        LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION} 
-        ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-        RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}) 
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
+# prevent pluginlib from using boost
+target_compile_definitions(${MOVEIT_LIB_NAME} PUBLIC "PLUGINLIB__DISABLE_BOOST_FUNCTIONS")
```

Comment:
Are these tags necessary now? Also, it's probably ok to remove the quotes and there is a double `_` in there